charts/evm-rollup/templates/statefulsets.yaml

```
@@ -67,7 +67,11 @@ spec:
           command: [ "geth" ]
           args:
             {{- range $arg := .Values.config.geth.flags }}
-            {{- if $arg.condition | default true }}
+            {{- if hasKey $arg "condition" }}
+            {{- if eq (tpl $arg.condition $) "true" }}
+            - --{{ $arg.name }}{{ if $arg.value }}={{ tpl $arg.value $ }}{{ end }}
+            {{- end }}
+            {{- else }}
```

We can probably keep this duplication to a minimum, can do same in the exec script?

If there is no condition set or the condition evaluates to true

```suggestion
            {{- $noCondition := not (hasKey $arg "condition") }}
            {{- if or ($noCondition) (eq (tpl $arg.condition $) "true") }}
```

---

Updated both statefulset and exec script with the new format, tested with and without metrics enabled using:
` helm template my-chart ./charts/evm-stack -f ./dev/values/rollup/dev.yaml`